Makefile

```diff
@@ -33,7 +33,9 @@ MAP     = gophermap
 
 INETD   = /etc/inetd.conf
 XINETD  = /etc/xinetd.d
-INETLIN = "gopher	stream	tcp	nowait	nobody	$(SBINDIR)/$(BINARY)	$(BINARY) -h `hostname`"
+# get OPTIONS line from gophernicus.env and use that also for inetd
+INETOPT = `grep '^OPTIONS=' $(NAME).env | tail -n 1 | sed -e 's/OPTIONS="*//;s/"*$$//'`
+INETLIN = "gopher stream tcp nowait nobody $(SBINDIR)/$(BINARY) $(BINARY) $(INETOPT)"
```

Comment:
What's the difference between this and what we had before? I'm not very familiar with inetd.

Reply:
It's basically the same, except that `INETOPT` is set to the contents of the `OPTIONS` variable in `gophernicus.env` i.e there is only one place for the default options. The whole addition basically just simplifies maintenance.